src/follower.py

```diff
@@ -73,21 +72,53 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
+    """
+    Follows the tag by panning camera towards the same direction as the wheels.
+    """
+    def pan_camera(self):
+        if self._tag._tag_data['x'] < 0:
```

Comment:
I don't think using the x value is a good way to do this. the x value is how far the tag is from the camera on the x axis. Using the direction property might make more sense for your case since we're looking to figure out how much should the camera pan by

Reply:
used direction property